test/index.js

```diff
@@ -678,6 +678,19 @@ test('chainRec', function() {
   eq(Z.chainRec.length, 3);
   eq(Z.chainRec.name, 'chainRec');
 
+  eq(
+    Z.chainRec(
+      Function,
+      function(next, done, n) {
+        return n === 30000
+          ? Z.map(done, function(env) { return n + env.inc; })
+          : Z.map(next, function(env) { return n + env.step; });
+      },
+      0
+    )({step: 2, inc: 100}),
+    30100
+  );
+
```

Comment:
I'd like to name the anonymous function (as we did in the case of `squash` below) so we can write:

```javascript
  eq(Z.chainRec(Function, TK, 0)({step: 2, inc: 100}), 30100);
```

`TK` is a placeholder name. I've no idea what the function should be named. ;)